### PR TITLE
Fix clear filters button of imaging uploader

### PR DESCRIPTION
**Pull request by cmadjar**
This pull request fixes the "Clear Filters" button that did not work on the imaging uploader (it was not mapped to any action).

See also: https://redmine.cbrain.mcgill.ca/issues/14721

Note: on the Redmine ticket, it is also mentioned than when clearing for example only CandID filter, row with empty CandID are not shown. This is fixed in a different PR: #3743

### CODE REVIEW

File: modules/imaging_uploader/jsx/ImagingUploader.js [hunk 1]
```diff
@@ -75,13 +80,14 @@ class ImagingUploader extends React.Component {
                 Module="imaging_uploader"
                 name="imaging_filter"
                 id="imaging_filter"
+                ref="imagingUploaderFilter"
```
Comment (davidblader) [inline modules/imaging_uploader/jsx/ImagingUploader.js]: string refs are being deprecated. use a callback function similar to [what was done here for data dictionary](https://github.com/aces/Loris/pull/3432/files#diff-b7cfdf269c10c4ab50f7710f6cfdaee7R81)

File: modules/imaging_uploader/jsx/ImagingUploader.js [hunk 2]
```diff
@@ -49,6 +50,10 @@ class ImagingUploader extends React.Component {
     this.setState({filter});
   }
 
+  resetFilters() {
+    this.refs.imagingUploaderFilter.clearFilter();
```
Comment (davidblader) [inline modules/imaging_uploader/jsx/ImagingUploader.js]: if using the ref callback method used below, you should remove the `.refs.` here

**Comment by cmadjar** [comment]
@davidblader Thank you! This is fixed.

**Comment by driusan** [comment]
Needs rebase.

**Comment by cmadjar** [comment]
@driusan PR has been rebased. Thanks!

**Pull request merged**